ConnectionManager.lib.nut

```diff
@@ -331,8 +377,13 @@ class ConnectionManager {
         }
 
         // Run the global onConnected Handler if it exists
-        if (_onConnect != null) {
-            imp.wakeup(0, function() { _onConnect(); }.bindenv(this));
+        if (_onConnect != null && _onConnect.len()) {
```

Comment:
This conditional is different than the others. Technically you don't have to check length, since you can loop through an empty table.

Reply:
Fixed